en/getting_started/README.md

```diff
@@ -0,0 +1,96 @@
+It is [dual-licensed under Apache 2.0 and GPLv3](https://github.com/mavlink/qgroundcontrol/blob/master/COPYING.md).
+
+To get the source files:
+1. Clone the repo (or your fork):
+   ```
+   git clone --recursive https://github.com/mavlink/qgroundcontrol.git
+   ```
+2. Update submodules (do this each time you pull new source code):
+   ```
+   git submodule update
+   ```
+
+> **Tip** Github source-code zip files cannot be used because these do not contain the appropriate submodule source code. You must use git!
+
+
+## Build QGroundControl
+
+### Native Builds
+
+*QGroundControl* builds are supported for OSX, Linux, Windows, iOS and Android. *QGroundControl* uses [Qt](http://www.qt.io) as its cross-platform support library and uses [QtCreator](http://doc.qt.io/qtcreator/index.html) as its default build environment.
+
+- **OSX:** OSX 10.7 or higher, 64 bit, clang compiler
+  > **Warning** XCode 8 requires a workaround described below.
+- **Ubuntu:** 64 bit, gcc compiler
+- **Windows:** Vista or higher, 32 bit, [Visual Studio 2015 compiler](http://www.visualstudio.com/downloads/download-visual-studio-vs#d-express-windows-desktop)
+- **iOS:** 8.0 and higher
+- **Android:** Jelly Bean (4.1) and higher. Standard QGC is built against ndk version 19.
+- <span id="qt_version"></span>**Qt version:** 5.9.3 **only**
+
+#### Install Qt
+
+You **need to install Qt as described below** instead of using pre-built packages from say, a Linux distribution, because *QGroundControl* needs access to private Qt headers.
+1. Download the [Qt installer](http://www.qt.io/download-open-source) for the [supported Qt version)[#qt_version]. You will also need to install the *Qt Speech* package.
+   - **Ubuntu:** Set the downloaded file to executable using:`chmod +x`. Install to default location for use with **./qgroundcontrol-start.sh.** If you install Qt to a non-default location you will need to modify **qgroundcontrol-start.sh** in order to run downloaded builds.
+   - **Windows:** Make sure to install *VS 2015 32 bit* package.
+
+1. Install Additional Packages (Platform Specific)
+   - **Ubuntu:** `sudo apt-get install speech-dispatcher libudev-dev libsdl2-dev`
+   - **Fedora:** `sudo dnf install speech-dispatcher SDL2-devel SDL2 systemd-devel`
+   - **Arch Linux:** `pacman -Sy speech-dispatcher`
+   - **Windows:** [USB Driver](http://www.pixhawk.org/firmware/downloads) to connect to Pixhawk/PX4Flow/3DR Radio
+   - **Android:** [Qt Android Setup](http://doc.qt.io/qt-5/androidgs.html)
+
+#### Building using Qt Creator
+
+1. Launch *Qt Creator* and open the **qgroundcontrol.pro** project.
+1. Select the appropriate kit for your needs:
+  - **OSX:** Desktop Qt 5.9.3 clang 64 bit
+  - **Ubuntu:** Desktop Qt 5.9.3 GCC bit
+  - **Windows:** Desktop Qt 5.9.3 MSVC2015 32bit
+  - **Android:** Android for armeabi-v7a (GCC 4.9, Qt 5.9.3)
+1. *Run Qmake* from the context menu for your target
```

Comment:
Is this correct. The previous instructions weren't clear.

Reply:
I haven't use QtCreator for a while so I don't know the precise sequence for running the build. I put in "*Run Qmake* from the context menu for your target" - is that enough for someone to work out how they run a build - or is there a better instruction - e.g.. Go to menu X, select Y

Reply:
That is meants for iOS builds only. I think it should be removed. My take on this is that the people should go to Qt web site docs to learn how to use Qt Creator to build for desktop or how to build iOS builds. Trying to duplicate too much of what is in Qt docs just leads to problems.

Reply:
OK, I have moved the warning/suggestion to use Xcode up to iOS and deleted the how to use QtCreator instructions. Cheers.